code/src/Components/LikePost.js

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from "react";
+
+export const LikePost = ({message}) => {
+
+  const [likes, setLikes] = useState(message.hearts)
+
+  useEffect(() => {
+
+     fetch(`https://technigo-thoughts.herokuapp.com/${message._id}/like`, {
+       method: "POST",
+          body: "", 
+       headers: { "content-type": "application/json",}
+       })
+  
+  }, []);
+  
+  return (
+    <>
+      <button className="emoji"  aria-label="heart emoji" 
+      onClick={() => setLikes(likes +1)} 
+      style={likes === 0 ? {backgroundColor: '#ccc9c9'} : {backgroundColor: '#ffb2b2'}}
+      >
```

Comment:
Suggestion of structure:
```suggestion
      <button 
        className="emoji"  
        aria-label="heart emoji" 
        onClick={() => setLikes(likes +1)} 
        style={likes === 0 ? {backgroundColor: '#ccc9c9'} : {backgroundColor: '#ffb2b2'}}
      >
```